Asterix/WSC_functions.py

```diff
@@ -143,67 +253,273 @@ def creatingInterractionmatrix(input_wavefront, testbed, dimEstim,
             pos_in_matrix += DM.basis_size
 
         else:
-            # Creating Interaction Matrix for thd DM if does not exist
-            init_pos_in_matrix = pos_in_matrix
-            print("")
-            print(fileDirectMatrix + " does not exists:")
+            # Finally we can measure the matrix if we
```

Comment:
je ne comprends pas cette phrase. "normalized with the testbed" = ?

Reply:
j'ai redeveloppé.